index.html

```diff
@@ -24,15 +24,118 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
+    </section>
+
+    <!-- Second project -->
+    <section class="work-page">
+        <img src="images/works/multi_post_story/cover.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
+            <li>Back</li>&nbsp;
+            <li>End</li>&nbsp;
+            <li>Dev</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
+            <li>2015</li>
+        </ul>
+        <p class="highlights">
+            A daily selection of privately personalized reads
+            no accounts for sign-ups required.
+        </p>
+        <ul class="tags">
+            <li><button>HTML</button></li>
+            <li><button>CSS</button></li>
+            <li><button>Js</button></li>
+        </ul>
+        <button class="project-detail">See Project</button>
+
+    <!-- Third project -->
+    </section>
+    <section class="work-page">
+        <img src="images/works/tonic/cover1.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
+            <li>Back</li>&nbsp;
+            <li>End</li>&nbsp;
+            <li>Dev</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
+            <li>2015</li>
+        </ul>
+        <p class="highlights">
+            A daily selection of privately personalized reads
+            no accounts for sign-ups required.
+        </p>
+        <ul class="tags">
+            <li><button>HTML</button></li>
+            <li><button>CSS</button></li>
+            <li><button>Js</button></li>
+        </ul>
+        <button class="project-detail">See Project</button>
+    </section>
+
+    <!-- About myself -->
+    <section class="about-myself">
+        <h1 class="about-myself-title">About <br> Myself</h1>
+        <p class="about-myself-content">
+            Hello I am a software develper. I can help you build a product,
+            feature or website. Look through some of my work and expereince.
+            If you like what you see and have a project you need coder, don't 
+            husitate to contact me.
+        </p>
+        <h3 class="links">LET'S CONNECT</h3>
             <ul class="socials">
                 <li><a href="#" class="fa fa-facebook"></a></li>
                 <li><a href="#" class="fa fa-twitter"></a></li>
                 <li><a href="#" class="fa fa-linkedin"></a></li>
                 <li><a href="#" class="fa fa-google"></a></li>
                 <li><a href="#" class="fa fa-github"></a></li>
+                <li><a href="#" class="fa fa-javascript"></a></li>
             </ul>
-        </main>
+        <button class="resume">Get My Resume</button>

```

Comment:
- [ ] Kindly change the background color of all your buttons  and the border around the button in the about section to match the Figma design 👍🏽 

FIgma Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198741235-00d99f1c-9a17-4986-95a4-2472088323c4.png)

YOur Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/198741401-aba28e4e-3e9c-4411-b69a-d5ac4c629730.png)

Reply:
Thanks for the tips. Correction made.